apps/civicsignalblog/src/payload/admin/scss/custom.scss

```diff
@@ -1,13 +0,0 @@
-@import "~payload/scss";
```

Comment:
Was this not being used before?

Reply:
Nope, I think I had added this one when I was experimenting with adding the custom select component for apps in the dashboard, but It's no longer relevant since I moved all the styling in the component itself.